plugin/core/windows.py

```diff
@@ -21,6 +21,7 @@
 from .workspace import ProjectFolders
 from .workspace import sorted_workspace_folders
 import threading
+from .message_request_handler import MessageRequestHandler
```

Comment:
Please keep this list of imports sorted (select all import lines -> Edit -> Sort Lines)